packages/functional-tests/pages/signupReact.ts

```diff
@@ -12,46 +12,67 @@ export class SignupReactPage extends BaseLayout {
     );
   }
 
+  get emailFirstHeader() {
```

Comment:
Is this used? I don't see any references to it when searching the code.

Reply:
Thank you, I'd added this when troubleshooting issues with the functional test but didn't need it in the end. Will remove.